src/app/components/dialog NewProject/DialogNewProject.tsx

```diff
@@ -1,57 +1,111 @@
 import * as React from 'react';
-import { Dialog, DialogActions, DialogContent, DialogTitle, Button, TextField } from '@mui/material';
-import { useForm, SubmitHandler } from 'react-hook-form';
+import { Dialog, DialogActions, DialogContent, DialogTitle, Button, TextField,
+  Select, MenuItem, FormControl, InputLabel 
+ } from '@mui/material';
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+import { QueryClient, useMutation, useQuery, useQueryClient} from 'react-query';
+import { createProject, getAllProjects } from '@/app/Service/ProjectServices';
+import { toast } from 'react-toastify';
+import { schemaCreateProject } from '@/app/Utils/ProjectValidation';
+import { yupResolver } from "@hookform/resolvers/yup";
+import { getLanguages } from '@/app/Service/LanguagesServices';
 
 interface IFormInput {
-  field1: string;
-  field2: string;
+  title:string;
+  targetLanguages:string[];
 }
 
 const DialogNewProject = () => {
+  const QueryClient = useQueryClient();
   const [open, setOpen] = React.useState(false);
-  
+  const { data: Allprojects} = useQuery('Allprojects', getAllProjects);
+  const { data: Languages } = useQuery('Languages', getLanguages);
```

Comment:
why you fetch this data but didn't use it ?

Reply:
i use it in invalidateQueries function  to Update data after create new project  => QueryClient.invalidateQueries('Allprojects');

Reply:
but this data didn't appear in the component , so why you fetch it or invalidate it?